docs/02-app/01-building-your-application/07-configuring/16-progressive-web-apps.mdx

```diff
@@ -437,14 +443,31 @@ export async function sendNotification(message: string) {
   }
 
   try {
+    // new:
     await webpush.sendNotification(
-      subscription,
+      {
+        endpoint: subscription.endpoint ?? '',
+        keys:  {
+          auth: subscription.keys?.auth ?? '',
+          p256dh: subscription.keys?.p256dh ?? ''
+        }
+      },
       JSON.stringify({
         title: 'Test Notification',
         body: message,
         icon: '/icon.png',
       })
     )
+    // old, but a type bug because PushSubscription from client is mixed up with webpush.PushSubscription which is required by webpush.sendNotification():
```

Comment:
If this is invalid we should remove it and not leave it around

Reply:
Yes in the final merge - absolutely. I thought this is a first proposal from myself to explain the problem. And therefore I wanted the old code to be visible. But true one can always see the diff...